packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -699,15 +742,26 @@ export class StatsAnalyzer extends EventsScope {
             `StatsAnalyzer:index#compareLastStatsResult --> No share RTP packets sent`,
             currentStats.totalPacketsSent
           );
-        } else {
+        } else if (this.lastEmittedStartStopEvent.share.local !== EVENTS.LOCAL_MEDIA_STOPPED) {
```

Comment:
same as above

Reply:
Moved this condition just above the event so that logs are printed every 5s and event is emitted only once.